spec/controllers/tournaments_controller_spec.rb

```diff
@@ -194,6 +194,29 @@
             expect(@group_stage_tournament.playoff_teams_amount)
               .to eq(create_group_tournament_data[:playoff_teams_amount])
           end
+
+          context 'playoff_teams_amount unacceptable' do
+            it 'is not a power of two' do
```

Comment:
this doesn't really read like a valid it-block.. they should read like actions and not input-conditions..

Reply:
so should I put a context around it saying context 'isn't a power of two' and the it part then saying 'it fails' ?

Reply:
That would be the "correct" way to do it, but in this case that would cause three contexts with just one it-block in them.. so it's probably fine to leave it like this or think of another way to structure this specific test

Reply:
is that better? 

https://github.com/turniere/turniere-backend/pull/55/commits/22040f7600f6d1c58936e98268b1da58910253e6